tools/prometheus.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/gogf/gf/v2/os/gtime"
```

Comment:
This isn't the correct gtime package :thinking: I was expecting the [Grafana plugin SDK's gtime package](https://pkg.go.dev/github.com/grafana/grafana-plugin-sdk-go@v0.277.1/backend/gtime#NewTimeRange). We should be able to use the `NewTimeRange` function and avoid any need for custom parsing ourselves.